src/statusBar/stusBarActionHandler.ts

```diff
@@ -21,9 +21,11 @@ export default async function handleStatus(
 ): Promise<void> {
   registerStatusHandlingCommand(status, context);
 
-  void setState({
-    [StatePayload.STATUS_SHOWN]: { text: status.message },
-  });
+  if (getStatusBarData()?.text !== status.message){
```

Comment:
Instead of exposing the statusBar (leaky abstraction) can we use something like `statusBarTextIs(status.message)`?

Reply:
👍